perun-db/postgres.sql

```diff
@@ -927,7 +927,7 @@ create table "tasks" (
 	status varchar(16) not null,        --state of task
 	start_time timestamp,                    --real start time of task
 	end_time timestamp,                      --real end time of task
-	engine_id integer not null, --identifier of engine which executing the task (engines.id)
+	engine_id integer, --identifier of engine which executing the task (engines.id)
```

Comment:
Please update DB version in a comment on first line of a file and at the end in initial version update. Thank you.